### DISCUSSION TITLE
What is the rate limit for POST /2/tweets(Client.create_tweet()) in Basic level?

### DISCUSSION THREAD

**Discussion by legendaryangelist** [tweepy/tweepy#2129] [Questions] [answered]
Hello.
Sorry for the question.
I saw the rate limit to create tweet in basic level is 5/15 min.
Is it right?

**Comment by Harmon758** [comment, marked as answer]
I'm not sure where you're seeing that, but both [the Developer Portal](https://developer.twitter.com/en/portal/products/basic) and [the documentation](https://developer.twitter.com/en/docs/twitter-api/rate-limits) say that it's 100 requests / 24 hours per user and 1667 requests / 24 hours per app.
